### ISSUE TITLE
Crash in uncompressing template

### ISSUE THREAD

**Issue by MarvinNazari** [carambalabs/pandora#2]
When i try to create 
`pandora create Kit marvin` 
i get
```
=> Downloading framework template
=> Framework template downloaded
=> Uncompressing template
/Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:528:in `rename': No such file or directory @ rb_file_s_rename - (/Users/marvinnazari/Code/template-master, /Users/marvinnazari/Code/Kit) (Errno::ENOENT)
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:528:in `block in mv'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:1571:in `block in fu_each_src_dest'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:1587:in `fu_each_src_dest0'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:1569:in `fu_each_src_dest'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/2.3.0/fileutils.rb:517:in `mv'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/pandora-frameworks-0.1.3/lib/pandora/commands/create.rb:59:in `unzip_template'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/pandora-frameworks-0.1.3/lib/pandora/commands/create.rb:30:in `execute'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/pandora-frameworks-0.1.3/lib/pandora/client/main.rb:10:in `create'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/pandora-frameworks-0.1.3/exe/pandora:5:in `<top (required)>'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/bin/pandora:23:in `load'
	from /Users/marvinnazari/.rbenv/versions/2.3.1/bin/pandora:23:in `<main>'

```

**Comment by MarvinNazari** [comment]
@pepibumur fixed in #3

**Comment by pepicrft** [comment]
thanks for the fix @MarvinNazari
